packages/sign-client/test/sdk/client.spec.ts

```diff
@@ -227,4 +229,33 @@ describe("Sign Client Integration", () => {
       await deleteClients(clients);
     });
   });
+
+  describe("session requests", () => {
+    it("should set custom request expiry", async () => {
+      const clients = await initTwoClients();
+      const {
+        sessionA: { topic },
+      } = await testConnectMethod(clients);
+
+      const expiry = 10000;
+
+      await Promise.all([
+        new Promise<void>((resolve) => {
+          clients.A.core.relayer.once(
+            RELAYER_EVENTS.publish,
```

Comment:
Was `RELAYER_EVENTS.publish` specifically added for hooking into here in the test or am I missing it being listened to in the source somewhere? (cmd+f on the diff only showed me this).

Reply:
This event was requested by the `Chat` team and was in the backlog to be added. I thought that was a good use case to utilize it in the expiry test and ship it

Reply:
Agreed in that case 👍 Thanks for context!